k8s-1.9.3/scripts/node01.sh

```diff
@@ -8,7 +8,7 @@ kind: MasterConfiguration
 apiServerExtraArgs:
   runtime-config: admissionregistration.k8s.io/v1alpha1
 token: abcdef.1234567890123456
-kubernetes-version: v1.9.3
+kubernetesVersion: v1.9.3
```

Comment:
The k8s-1.9.3 extra layer is there because of this line change (not a cluster rebuild, just copied the file over).